automation-tests/test/utilities.ts

```diff
@@ -0,0 +1,337 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { ChildProcess, exec } from 'child_process';
+import clipboard from 'clipboardy'
```

Comment:
we should probably create a ticket to follow up on the 3PP status of any imports that are net new in this project.